MarsRover/Rover.cs

```diff
@@ -17,6 +16,11 @@ public Rover(Direction direction, int x, int y)
 
         public Location Location { get; private set; }
 
+        public Location SetLocation(Location location) // TODO: valid way to do this? name ok? Not sure it works...
+        {
+            return new Location(location.X, location.Y);

```

Comment:
`Location = location;`, but you could also just have a public setter on the `Location` prop if you were to need this functionality